wgpu-types/src/lib.rs

```diff
@@ -1786,7 +1786,7 @@ impl<L> TextureViewDescriptor<L> {
     }
 
     pub fn level_count(&mut self, level_count: u32) -> &mut Self {
-        self.level_count = Some(level_count);
+        self.level_count = Some(NonZeroU32::new(level_count).unwrap());
```

Comment:
now that I see it, I wonder if we should just do `level_count = NonZero::U32(...)`

Reply:
We would be checking for 0 then. Or should we add a separate `if` for that?

Reply:
yeah, I mean, it's a builder, and passing 0 explicitly is a weird case
anyhow, we can keep that in mind, no need to do these changes, and there isn't an agreement on it yet :)